biogrid/libs/biogrid-simulator/src/lib/biobattery/biobattery-class.ts

```diff
@@ -11,7 +11,7 @@ import { SMALL_BATTERY } from '../config';
 export class BioBattery implements Battery {
   private energyInJoules: Energy;
   private readonly maxCapacity: Energy;
-  name: string;
+  gridItemName: string;
```

Comment:
leave a comment explaining that this should include the type as part of the name?

Reply:
I will address it before merge